Apps/W1/Shopify/app/src/Order handling/Tables/ShpfyOrderTaxLine.Table.al

```diff
@@ -44,6 +44,21 @@ table 30122 "Shpfy Order Tax Line"
             Caption = 'Currency Code';
             DataClassification = SystemMetadata;
             Editable = false;
+            ObsoleteState = Pending;
+            ObsoleteTag = '21.3.0.0';
+            ObsoleteReason = 'Is available in Shopify Order Header table';
+        }
+        field(7; "Presentment Amount"; Decimal)
```

Comment:
is Presentment really the correct term?

Reply:
Presentment is the correct name, this is the name used in Shopify.

Reply:
👍